vars/rollingUpgradePipeline.groovy

```diff
@@ -204,6 +204,8 @@ def call(Map pipelineParams) {
                             def base_version = version
                             params_mapping[base_version] = params.collectEntries { param -> [param.key, param.value] }
                             params_mapping[base_version].put('scylla_version', base_version)
+                            // since scylla-pkg might pass this one, we are not supporting it here, as we always starts by version
+                            params_mapping[base_version].remove('scylla_repo')
```

Comment:
Will it work if the `scylla_repo` is absent?

Reply:
> To remove an entry from the map, use the remove() method. It returns the value that was previously associated with the key passed in, otherwise it returns null if it did not find the given key value in the map to remove.

https://docs.oracle.com/en/cloud/saas/applications-common/25a/cgsac/working-with-maps.html